src/main/java/ru/sbtqa/tag/apifactory/rest/RestEntityImpl.java

```
@@ -88,43 +93,24 @@ public Bullet post(String url, Map<String, String> headers, Object body) throws
         try {
             LOG.info("Sending 'POST' request to URL : {}", url);
             final HttpPost post = new HttpPost(url);
+            return performRequest(headers, body, client, post);
+        } catch (IOException ex) {
+            LOG.error("Failed to get response", ex);
+        } finally {
+            HttpClientUtils.closeQuietly(client);
+        }
 
-            for (Map.Entry<String,String> h: headers.entrySet()) {
-                post.setHeader(h.getKey(), h.getValue());
-            }
-
-            LOG.info("Headers are: {}", headers);
-
-            List<NameValuePair> postParams = new ArrayList<>();
-            String encoding = Props.get("api.encoding");
-            if (body instanceof Map) {
-                Map<String, String> params = (Map<String, String>) body;
-
-                for (Map.Entry<String,String> e: params.entrySet()) {
-                    postParams.add(new BasicNameValuePair(e.getKey(), e.getValue()));
-                }
-
-                if (!postParams.isEmpty()) {
-                    post.setEntity(new UrlEncodedFormEntity(postParams));
-                }
-                LOG.info("Body (form-data) is: {}", body);
-            } else if (body instanceof String) {
-                post.setEntity(new StringEntity((String) body, encoding));
-            }
-
-            HttpResponse response = client.execute(post);
+        return null;
```

Maybe throw ApiRestException or something instead of null returning?